src/content_sao.cpp

```diff
@@ -1349,6 +1358,38 @@ void PlayerSAO::setBreath(const u16 breath, bool send)
 		m_env->getGameDef()->SendPlayerBreath(this);
 }
 
+void PlayerSAO::hide()
+{
+	if (m_hidden)
+		return;
+
+	m_hidden = true;
+	if (! m_player)
```

Comment:
```suggestion
	if (!m_player)
```

Please also remove spaces between `!` and variable name everywhere else within the changes too.